### PR TITLE
FUSETOOLS-3480: Move JDK8 logic into utils and reuse in all relevant …

### PR DESCRIPTION
**Pull request by pospisilf**
…UI tests

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by djelinek** [comment]
testPR

**Comment by apupier** [comment]
need to rebase on master to have green build on both PR jobs

**Comment by pospisilf** [comment]
rebased and signed off

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/JDK8Check.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
```
Comment (apupier) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/JDK8Check.java:2]: 2017? Is it coming from another place?

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/JDK8Check.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ * Checks, if JDK8 is available.
+ * 
+ * @author djelinek, fpospisi
+ */
+public class JDK8Check {
+
+	public static final String JDK_WARNING_MESSAGE = "No Strictly compliant JRE detected";
+
+	/*
+	 * Checks, if JDK8 is available in instance of CRS.
```
Comment (apupier) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/JDK8Check.java:32]: CRS means CodeReady Studio?

This code is also used in Fuse Tooling only. What about avoiding the acronym and replace with a more generic indication?

**Review by apupier — COMMENTED** [review]
Good to have separated the code related to JDK8

2 comments seems incorrect/improvable

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/JDK8Check.java (hunk 1 above)
Comment (pospisilf) [reply uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/JDK8Check.java:2]: Yes, it's taken from Project Factory.

**Comment by pospisilf** [comment]
Both comments hopefully improved.

**Review by apupier — APPROVED** [review]

**Review by djelinek — APPROVED** [review]

**Comment by djelinek** [comment]
testPR

**Comment by djelinek** [comment]
testPR

**Comment by djelinek** [comment]
testPR

**Comment by djelinek** [comment]
testPR

**Comment by djelinek** [comment]
testPR

**Pull request merged**